mongo_connector/__init__.py

```diff
@@ -11,7 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import logging
+
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
 
-__version__ = '2.5.0.dev0'
+version_info = (2, 5, 0, 'dev0')
+version = '.'.join(str(v) for v in version_info)
+__version__ = version
```

Comment:
The reason it worked is because I moved `__version__` in #582. I'll rebase this PR and move the logging stuff out of `__init__.py` too. Yeah, we don't need to change the DocManager structure but need to be aware that they all share the same directory when installed so they can't have any files with the same name and `__init.py` should never be modified.